api/fixtures/example.go

```diff
@@ -340,6 +356,46 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 					IOPS: o.AWS.RootVolumeIOPS,
 				},
 			}
+		case hyperv1.KubevirtPlatform:
+			// TODO nargaman
+			// As long as there is no official container image
+			// The image must be provided by user
+			// Otherwise it must fail
+			// https://issues.redhat.com/plugins/servlet/mobile#issue/RFE-2501
+			if o.Kubevirt.Image == "" {
+				return nil, errors.New("The image for the Kubevirt machine must be provided by user (\"--kubevirt-image\" flag")
```

Comment:
I think we expect that the commandline part already ensured that this value is populated. In fact I think that you are doing this alreay. You can probably remove this check and simplify the function return signature again.

Reply:
Its can't be validated in the command line, because there is option to create the cluster without nodepool and create the nodepool later, in this case this parameter can be empty

Reply:
But can't you then outside already verify that if the creation is requested, that you are also requiring this to be filled? 

For example at https://github.com/openshift/hypershift/pull/779/files#diff-a8a4a70fd459a31d15b6500359d7e0e3cd47c5f7a48f551071756ddc395eaa20R55 ?

Reply:
I just want to ensure that we keep the input validation together and don't spread it too much. The entrypoint above is supposed to see all persistent values filled already and should be able to handle your case based on the available info.